### PR TITLE
DHFPROD-7309: Match Keyboard Accessibility

### PR DESCRIPTION
**Pull request by littlecherro**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

**Review by amejiaq — APPROVED** [review]
Pretty solution for the unit test!

**Comment by littlecherro** [comment]
Notes: 
The select component on Values to Ignore doesn't behaves as expected. There's going to be a ticket for that.
Also, there's an issue with TreeSelect component where is almost impossible to reach the styles when its focused, so a workaround was implemented for that.

**Review by briantang — CHANGES_REQUESTED** [review]
Screenshot 2023-02-09 at 10 21 47 AM

I can't seem to select the other radio options "Test URIs with All Data" and "Test All Data"

File: marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.test.tsx [hunk 1]
```diff
@@ -139,4 +139,72 @@ describe("Matching Step Detail view component", () => {
     expect(getByLabelText("matchCombinationsHeading")).toBeInTheDocument();
     expect(getByLabelText("noMatchedCombinations")).toBeInTheDocument();
   });
+
+  it.only("Keyboard Navigation sequence is correct", async () => {
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.test.tsx:143]: May have forgotten to remove the "only" here

**Comment by littlecherro** [comment]
> Screenshot 2023-02-09 at 10 21 47 AM
> 
> I can't seem to select the other radio options "Test URIs with All Data" and "Test All Data"

try with the arrows :) 

image

**Comment by littlecherro** [comment]
> Screenshot 2023-02-09 at 10 21 47 AM
> 
> I can't seem to select the other radio options "Test URIs with All Data" and "Test All Data"

@briantang try the arrows 🤔
image

**Comment by briantang** [comment]
> > Screenshot 2023-02-09 at 10 21 47 AM
> > I can't seem to select the other radio options "Test URIs with All Data" and "Test All Data"
> 
> @briantang try the arrows 🤔 image

Oops right I see that working now.

**Review by briantang — APPROVED** [review]

**Review by amejiaq — APPROVED** [review]

**Review by amejiaq — DISMISSED** [review]

**Review by briantang — APPROVED** [review]

**Review by amejiaq — APPROVED** [review]

**Pull request merged**